swagger.json

```diff
@@ -1768,6 +1768,43 @@
                     }
                 }
             }
+        },
+        "/fleet/assets/{asset_id}/location": {
+            "get": {
+                "tags": [
+                    "Default", "Fleet", "Assets"
+                ],
+                "summary": "/fleet/assets/{assetId:[0-9]+}/location",
+                "description": "Fetch the historical locations for the asset.",
+                "operationId": "GetAssetLocation",
+                "parameters": [
+                    { "$ref": "#/parameters/accessTokenParam" },
+                    {
+                        "name": "asset_id",
```

Comment:
this should probably be a `$ref` in `#/parameters/` since we'll use it a bunch

Reply:
is this basically making a variable parameter? i'll try my best swagger skills but don't have high hopes

Reply:
done